manual/manual/refman/exten.etex

```diff
@@ -1315,6 +1315,53 @@ to implement dynamic types.
           | None -> None
           | Some Eq -> Some x
 \end{verbatim}
+\paragraph{Type variable names in error messages}
+The typing of pattern matching in presence of GADT can generate many
+existential types. When reporting a typing error, some of these
+existential types can be exposed and therefore need a
```

Comment:
I'm not fond of "can be exposed and therefore need a compiler-generated name", as "exposed" does not mean much.  => Maybe "may need to be referred to" instead of "can be exposed", or even "may need to be named in the error explanation" for the whole thing?